src/App.js

```diff
@@ -33,11 +33,13 @@ class App extends Component {
             <HashRouter basename='/'>
                 <div className="App">
                     <header className="App-header">
-                        <Route exact path="/login">
-                            {loginScreen()}
-                        </Route>
-                        <Route path="/" component={AuthenticatedApp}/>
-                        <Route path="/gamelist" component={JackboxGameList}/>
+                        <Switch>
+                            <Route exact path="/login">
+                                {loginScreen()}
+                            </Route>
+                            <Route exact path="/gamelist" component={JackboxGameList}/>
```

Comment:
yesssss the /gamelist page has been bugging me for so long now

Reply:
Was just missing a `<Switch>` to get it working properly